src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const elements = [...document.querySelectorAll('span.population')];
+
+const populationArr = elements.map((element) => {
+  return Number(element.innerText.split(',').join(''));
+});
+
+const totalPopulation = populationArr.reduce((acc, oneCountryPopulation) => {
+  return acc + oneCountryPopulation;
+}, 0);
+
+const averagePopulation = totalPopulation / populationArr.length;
+
+document.querySelector('span.total-population').innerText =
+  totalPopulation.toLocaleString('en-US');
+
+document.querySelector('span.average-population').innerText =
+  averagePopulation.toLocaleString('en-US');
```

Comment:
Same comment as above for the `toLocaleString` method. It's better to use the user's locale.